Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift

```diff
@@ -1,85 +1,82 @@
 //
-//  ViewController.swift
-//  ThumbnailExample
+//  BannerManager.swift
+//  Direct Integration Sample
 //
-//  Copyright © 2020 Ogury Co. All rights reserved.
+//  Created by Fernand Peng on 09/11/2021.
 //
 
 import UIKit
 import OguryChoiceManager
 import OguryAds
 
-class ViewController: UIViewController {
+class BannerViewController: UIViewController {
     
     @IBOutlet weak var statusTextView: UITextView!
     @IBOutlet weak var mpuView: UIView!
     @IBOutlet weak var smallBannerView: UIView!
-    
-    var mpuLoaded: Bool = false
-    var smallBannerLoaded: Bool = false
-    
+
     var smallBanner: OguryAdsBanner?
     var mpuBanner: OguryAdsBanner?
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.addNewStatus("Choice Manager Loading...")
+        self.addNewStatus("[Choice Manager] Loading...")
         
         //The setup of Ogury Choice Manager and Ogury Ads is done AppDelegate.swift file.
         OguryChoiceManager.shared().ask(with: self) { (error, answer) in
             if error == nil {
                 switch answer {
                 case .noAnswer: // TCF Option
-                    self.addNewStatus("Choice Manager No Answer")
+                    self.addNewStatus("[Choice Manager] No Answer")
                 case .fullApproval: // TCF Option
-                    self.addNewStatus("Choice Manager Full Approval")
+                    self.addNewStatus("[Choice Manager] Full Approval")
                 case .partialApproval: // TCF Option
-                    self.addNewStatus("Choice Manager Partial Approval")
+                    self.addNewStatus("[Choice Manager] Partial Approval")
                 case .refusal: // TCF Option
-                    self.addNewStatus("Choice Manager Refusal")
+                    self.addNewStatus("[Choice Manager] Refusal")
                 case .saleAllowed: // CCPA Option
-                    self.addNewStatus("Choice Manager Sale Allowed")
+                    self.addNewStatus("[Choice Manager] Sale Allowed")
                 case .saleDenied: // CCPA Option
-                    self.addNewStatus("Choice Manager Sale Denided")
+                    self.addNewStatus("[Choice Manager] Sale Denided")
                 default:
-                    self.addNewStatus("Choice Manager Unknown Option")
+                    self.addNewStatus("[Choice Manager] Unknown Option")
                 }
             } else {
-                self.addNewStatus("Choice Manager error : \(error.debugDescription)")
+                self.addNewStatus("[Choice Manager] error : \(error.debugDescription)")
             }
         }
     }
 
     @IBAction func loadMPUBtnPressed(_ sender: Any) {
-        self.addNewStatus("MPU Loading ...")
-        mpuBanner = OguryAdsBanner.init(adUnitID: "ogury_adunit")
+        addNewStatus("[Ad][MPU] Loading ...")
+        mpuBanner = OguryAdsBanner.init(adUnitID: ConstantKeys.mpuAdUnit)
```

Comment:
```suggestion
        mpuBanner = OguryAdsBanner(adUnitID: ConstantKeys.mpuAdUnit)
```

Reply:
.init is not needed in Swift

Reply:
Updated in 750d11cae96c04f3272b4a34935c4ea6ed56a689